torch/csrc/cudnn/cuDNN.cwrap

```diff
@@ -5,6 +5,8 @@
 
 #include <Python.h>
 #include "cudnn-wrapper.h"
+#include "DynamicTypes.h"
+#include "../DynamicTypes.h"
```

Comment:
Are there two DynamicTypes.h files?

Reply:
Yes, this PR adds a new one in `cudnn` dir

Reply:
Happy to rename it, I just picked the first name that came to mind. I'm hoping to delete it entirely when this code moves into ATen.